README.md

```diff
@@ -42,16 +42,16 @@ Motia is a **modern backend framework** that unifies APIs, background jobs, even
 
 Motia brings cohesion to the fragmented backend world with our core primitive: the **Step**. Think of Steps like React Components, but for backends.
 
-![Motia combines APIs, background queues, and AI agents into one system](https://github.com/MotiaDev/motia/blob/main/assets/final.gif?raw=true)
+![Motia combines APIs, background queues, and AI agents into one system](/assets/motia-architecture-with-bg.png)
 
 ### 🧱 The Step Philosophy
 
-- **🎯 Single Purpose**: Each Step performs one task
-- **🌍 Language agnostic**: Each Step can be in a different language while being part of the same workflow
-- **⚡ Versatile**: Steps can trigger APIs, background jobs, and AI Agents
-- **👁️ Observable**: Everything is observable by default
-- **🌊 Workflows**: Collections of connected steps that form complete processes
-- **🏪 State Management**: Shared state across all steps with full traceability
+- **🎯 Your Logic, Your Step**: A Step holds your business logic. It can be a simple function, a call to a database, or a complex AI agent. This is where your application's real work gets done.
+- **🌍 Any Language, One Workflow**: Write Steps in TypeScript, Python, etc. all in the same project. Use Python for your AI agents and TypeScript for your API, and Motia makes them work together effortlessly.
```

Comment:
Suggestion: etc may sound we support others, I think it's better to show we have plans to add other languages instead

```suggestion
- **🌍 Any Language, One Workflow**: Write Steps in TypeScript, Python, and other languages to come. all in the same project. Use Python for your AI agents and TypeScript for your API, and Motia makes them work together effortlessly.
```